presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/TestPickTableLayout.java

```diff
@@ -46,13 +49,15 @@
             new TpchTableLayoutHandle(new TpchTableHandle("local", "nation", 1.0), Optional.empty()));
 
     private RuleSet pickTableLayout;
+    private RuleSet pickTableLayoutReplaceExisting;
     private TableHandle nationTableHandle;
     private TableLayoutHandle nationTableLayoutHandle;
 
     @BeforeMethod
     public void setUpPerMethod()
     {
-        pickTableLayout = new PickTableLayout(tester().getMetadata());
+        pickTableLayout = new PickTableLayout(tester().getMetadata(), new SqlParser(), false);
+        pickTableLayoutReplaceExisting = new PickTableLayout(tester().getMetadata(), new SqlParser(), true);
```

Comment:
TPCH connector does support layouts for `orderstatus` column in `orders` table. Therefore you can add proper tests with connector accepting predicate fully, see: https://github.com/prestodb/presto/pull/8865/commits/da183f4b4300c708a4f321173c88d72594687b90 and https://github.com/prestodb/presto/pull/8865/commits/e0f2750bcbe6aedb16cca559ab3727cc8853826f